src/intro.md

```diff
@@ -1,35 +1,34 @@
 
 True data provenance will make data truly portable and allow users to share it with others as they see fit.
 
 ## How Does the TLSNotary Protocol Work?
 
 The TLSNotary protocol consists of 4 steps:
-1. The `User` **requests** the data from the server in a secure and private way
-2. The Notary **checks** that the user did not tamper with the data and **signs** the transcript
-3. The `User` **selective discloses** the data to the `Verifier`
-4. The `Verifier` **verifies** the data
+1. The `User` **requests** the data from the server privately and securely.
+2. The Notary **checks** for data tampering and **signs** the transcript.
+3. The `User` **selectively discloses** the data to the `Verifier`.
+4. The `Verifier` **verifies** the data.
 
 ![](./png-diagrams/overview3.png)
 
 ### ① Multi-party TLS Request
 
-TLSNotary works by adding a third party, the `Notary`, to the usual TLS connection between the `User` and a Server. This `Notary` is **not an intermediary**. Instead, the `Notary` participates in a **secure multi-party computation** (MPC) to jointly operate the TLS connection without ever seeing the data in plain text; the `Notary` only sees encrypted data. Moreover, because the `Notary` only sees the ephemeral keys of the Server, the `Notary` does not know which Server the `User` is communicating with. The TLSNotary protocol is transparent to the Server. From the Server's perspective, the TLS connection is the same as all other connections. Therefore, no modifications to the TLS protocol are required.
+TLSNotary works by adding a third party, the `Notary`, to the usual TLS connection between the `User` and a Server. This `Notary` is **not an intermediary**. Instead, the `Notary` participates in a **secure multi-party computation** (MPC) to jointly operate the TLS connection without ever seeing the data in plain text; the `Notary` only sees encrypted data. Given that the `Notary` only sees the ephemeral keys of the Server, the `Notary` does not know which Server the `User` is communicating with. The TLSNotary protocol is transparent to the Server. From the Server's perspective, the TLS connection is the same as all other connections. Therefore, no modifications to the TLS protocol are needed.
 
-[- Transport Layer Security (TLS)
+<!-- - Transport Layer Security (TLS)
     - Encryption: hides data from third parties
     - Authentication: ensures that the parties exchanging information are who they claim to be
-    - Integrity: verifies that data has not been forged or tampered with]: #
-
+    - Integrity: verifies that data has not been forged or tampered with -->
 ### ② Notarization
 
-The `User` forwards the encrypted TLS traffic to the `Notary`, which checks that it has not been tampered with and notarizes the whole TLS session by signing a transcript of it.
+The `User` forwards the encrypted TLS traffic to the `Notary`, which validates its integrity and then notarizes the entire TLS session by signing its transcript.
```

Comment:
In light of what I suggested above, Id rephrase (rough draft without careful word choice)

The `User` engages in MPC with the `Notary` to perform all cryptographic operations on the TLS traffic. Thanks to the properties of MPC, the Notary never learns the data which he signs and yet he is able to validate the authenticity of the data.

Reply:
Thanks for comments. I have rephrased this entire paragraph.